GitTfs/Core/GitChangeInfo.cs

```diff
@@ -68,30 +68,25 @@ public static IEnumerable<GitChangeInfo> GetChangedFiles(TextReader reader)
         /// <returns>changes once filtered</returns>
         private static IEnumerable<GitChangeInfo> FilterChangesIntroducedByCaseRenaming(IEnumerable<GitChangeInfo> changes)
         {
-            //filter out "case only renames" because they cannot be represented in TFS
-            var remainingChanges = changes.Where(change => change.Status != ChangeType.RENAMEEDIT ||
-                String.Compare(change.path, change.pathTo, StringComparison.OrdinalIgnoreCase) != 0 ||
-                change.newSha != change.oldSha).ToList();
```

Comment:
This is not a functional change. The changes which were filtered here, are now filtered in 
``change.Status = change.newSha != change.oldSha ? ChangeType.MODIFY : ElementToRemove;``